conf4j-core/src/main/java/com/sabre/oss/conf4j/converter/standard/ByteTypeConverter.java

```diff
@@ -24,18 +24,15 @@
 
 package com.sabre.oss.conf4j.converter.standard;
 
-import com.sabre.oss.conf4j.converter.TypeConverter;
-
 import java.lang.reflect.Type;
 import java.util.Map;
-import java.util.Objects;
 
 import static java.util.Objects.requireNonNull;
 
 /**
  * This class converts {@link Byte} to/from string.
  */
-public class ByteTypeConverter implements TypeConverter<Byte> {
+public class ByteTypeConverter extends FormattedTypeConverter<Byte> {
```

Comment:
This comment applies to all number related converters. It would be much better to use _NumberFormat_ class for formatting because it supports parsing too. In conf4j in most cases converters convert a string to a type (not type value to string). For that reason conversion from string is most important. _NumberFormat_ class provides enough functionality required for formatting numbers. Because parsing the format may be slow, consider caching _NumberFormat_ (I would recommend creating some performance tests to verify it is necessary). In case the cache will be used, make sure only one thread can access NumberFormat at the same time.